PWGCF/MultiparticleCorrelations/Core/MuPa-MemberFunctions.h

```diff
@@ -5519,7 +6137,7 @@ void InternalValidation()
     // For this option, vn's and psin's are constant for all simulated events, therefore I can configure fPhiPDF outside of loop over events.
     // Remark: The last parameter [18] is a random reaction plane, keep in sync with fPhiPDF->SetParameter(18,fReactionPlane); below
     //         Keep also in sync with const int gMaxHarmonic = 9; in *GlobalConstants.h
-    fPhiPDF = new TF1("fPhiPDF", "1 + 2.*[0]*TMath::Cos(x-[1]-[18]) + 2.*[2]*TMath::Cos(2.*(x-[3]-[18])) + 2.*[4]*TMath::Cos(3.*(x-[5]-[18])) + 2.*[6]*TMath::Cos(4.*(x-[7]-[18])) + 2.*[8]*TMath::Cos(5.*(x-[9]-[18])) + 2.*[10]*TMath::Cos(6.*(x-[11]-[18])) + 2.*[12]*TMath::Cos(7.*(x-[13]-[18])) + 2.*[14]*TMath::Cos(8.*(x-[15]-[18])) + 2.*[16]*TMath::Cos(9.*(x-[17]-[18]))", 0., o2::constants::math::TwoPI);
+    fPhiPDF = new TF1("fPhiPDF", "1 + 2.*[0]*cos(x-[1]-[18]) + 2.*[2]*cos(2.*(x-[3]-[18])) + 2.*[4]*cos(3.*(x-[5]-[18])) + 2.*[6]*cos(4.*(x-[7]-[18])) + 2.*[8]*cos(5.*(x-[9]-[18])) + 2.*[10]*cos(6.*(x-[11]-[18])) + 2.*[12]*cos(7.*(x-[13]-[18])) + 2.*[14]*cos(8.*(x-[15]-[18])) + 2.*[16]*cos(9.*(x-[17]-[18]))", 0., o2::constants::math::TwoPI);
```

Comment:
In here, being `TF1` a `ROOT` object I would keep `TMath::Cos` which is predictable
I guess you have used `std::cos` at some point and probably it did not work
I am not quite sure what `ROOT` will decide to use for `cos` and that is why I would adhere to what is predictable 
Just consider it for the next iteration

Reply:
@vkucera, what is your view on this?

Reply:
AFAIK, `TMath::Cos` and `std::cos` are identical both in plain code (see https://root.cern.ch/root/html524/TMath.html#TMath:Cos) and in `TF1` definition:
```c++
auto f1 = new TF1("f1", "std::cos(x)", 0, 6.28);
auto f2 = new TF1("f2", "TMath::Cos(x)", 0, 6.28);
auto c1 = new TCanvas("c1");
f1->SetLineWidth(4);
f1->SetLineColor(1);
f1->Draw();
f2->Draw("same");
```

Reply:
I cross-checked only that in TF1 formula "TMath::Cos" and "cos" give the same results, I didn't check "std::cos" in this context. 

But I see now that I can also use "std::cos" in TF1 fearlessly, therefore I will take that road in the next commit.